modules/bookCollection.js

```diff
@@ -0,0 +1,54 @@
+class BookCollection {
+  constructor(bookArray) {
+    this.bookArray = bookArray;
+  }
+
+  displayBooks() {
+    const bookContainer = document.querySelector('.books-container');
+    bookContainer.innerHTML = '';
+    this.bookArray.forEach((book) => {
+      bookContainer.insertAdjacentHTML(
+        'beforeend',
+        `
+        <div class="book-details" id="book-${book.id}">
+          <p>"${book.title}" by ${book.author}</p> 
+          <button class="remove" type="button" id="remove-${book.id}">Remove</button>
+        </div>`,
+      );
+    });
+
+    if (this.bookArray.length !== 0) {
+      bookContainer.style.border = 'solid 3px #000000';
+    } else bookContainer.style.border = 'none';
+  }
+
+  storeInLocalBrowser() {
+    localStorage.setItem('bookCollectionArray', JSON.stringify(this.bookArray));
+  }
+
+  removeBook(book) {
+    this.bookArray.splice(book, 1);
+    this.storeInLocalBrowser();
+    this.bookArray.forEach((book, i) => {
+      book.id = i;
+    });
+    this.displayBooks();
+  }
+
+  addBook(titleInput, authorInput) {
+    if (titleInput.value && authorInput.value) {
+      const title = titleInput.value;
+      const author = authorInput.value;
+      this.bookArray.push({
+        title,
+        author,
+        id: this.bookArray.length,
+      });
```

Comment:
- [ optional ] great job in creating the addBook method. I'll just recommend using a different approach for the id. By using the array length you may end up having multiple books with the same id which Is not convenient. A different approach could be to use the date string since its more unique `new Date.now()`